sdk/src/main/scala/io/horizen/consensus/package.scala

```diff
@@ -93,17 +93,25 @@ package object consensus {
     new BigInteger(1, vrfOutput.bytes())
   }
 
-  def vrfProofCheckAgainstStake(vrfOutput: VrfOutput, actualStake: Long, totalStake: Long, stakePercentageFork: Boolean): Boolean = {
+  def vrfProofCheckAgainstStake(vrfOutput: VrfOutput, actualStake: Long, totalStake: Long, stakePercentageFork: Boolean, activeSlotCoefficient: Double): Boolean = {
     val requiredStakePercentage: BigDecimal = vrfOutputToRequiredStakePercentage(vrfOutput, stakePercentageFork)
     val actualStakePercentage: BigDecimal = new BigDecimal(actualStake).divide(new BigDecimal(totalStake), stakeConsensusDivideMathContext)
 
-    requiredStakePercentage.compareTo(actualStakePercentage) match {
-      case -1 => true //required percentage is less than actual
-      case  0 => true //required percentage is equal to actual
-      case  _ => false //any other case
+    if (activeSlotCoefficient >= 0) {
+      val actualStakePercentageWithActiveSlotCoefficient = 1 - Math.pow( 1-activeSlotCoefficient, actualStakePercentage.doubleValue())
```

Comment:
Maybe I'm missing some kind of info but shouldn't it be the other way around? That is, the `actualStakePercentage` as base number of the `Math.pow` function and the `1-activeSlotCoefficient` as exponent. In this way we will have a value of `actualStakePercentage` corrected by the exponent of domain _[0:1]_

Reply:
The formula was changed accordingly to the analysis made by the Crypto Team. You can find the results here: https://horizenlabs.atlassian.net/browse/CRYPTO-479
Please tell me if you still have concerns.